lib/l10n/intl_en.arb

```diff
@@ -28,7 +28,7 @@
     "@apiKeyDialog_apiKeyIsNotSaved": {},
     "apiKeyDialog_removeApiKey": "Remove API key",
     "@apiKeyDialog_removeApiKey": {},
-    "apiKeyDialog_apiKeySaved": "✔️ Your API key has been successfully added",
```

Comment:
if you are removing emoji from snackbar messages, you should do it for all of them